src/interactive.rs

```diff
@@ -157,14 +157,14 @@ fn _check_json_limit(json: &serde_json::Value, path: &mut String, limit: usize)
                 return false;
             }
             for (idx, item) in items.iter().enumerate() {
-                write!(path, "[{}]", idx).expect("formatting succeeds");
+                write!(path, "[{}]", idx).expect("formatting failed");
```

Comment:
I'm not sure I understood what `expect()` does. Many of the messages in `expect()` calls seemed to be the opposite of what it seamed they should be :confused:

Reply:
It writes something like:
```
expectation failed: formatting failed
expectation failed: formatting succeeds
```
So generally both forms read good enough for me, and I see both forms in other's Rust code. Note: this is printed along with more cryptic stuff and (hopefully) a backtrace and is treated as "assertion failure". So these errors are always bugs, and end users aren't supposed to understand what's going on. This is only for us to grep for the issue more quickly (especially if the is no stack trace).